docs/roles/project-coordinator.rst

```diff
@@ -11,23 +11,23 @@ responsibilities and role of the project coordinator.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
-
+The project coordinator is responsible for ensuring that all club projects have the resources needed to continue developing. This may include monetary resources, connections to mentors or the like and infrastructure to host the projects. 
```

Comment:
As per the [Sphinx style guide](https://documentation-style-guide-sphinx.readthedocs.io/en/latest/index.html), lines should be limited to 79 characters per line. Please submit a new PR with the correct line lengths.